src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -238,15 +244,15 @@ private CaseDTO mapCaseContainerDTO(CaseContainerDTO caseDetails) {
     Optional<CachedCase> cachedCase = dataRepo.readCachedCaseByUPRN(uprn);
     if (cachedCase.isPresent()) {
       log.with("uprn", uprn).debug("Returning stored case details for UPRN");
-      return Collections.singletonList(caseDTOMapper.map(cachedCase.get(), CaseDTO.class));
+      return createNewCaseResponse(cachedCase.get());
     }
 
     // New Case
-    CaseDTO result = caseDTOMapper.map(createNewCase(uprn.getValue()), CaseDTO.class);
+    CachedCase newcase = createNewCase(uprn.getValue());
```

Comment:
createNewCacheCase() would be more descriptive

Reply:
Done.